rust/src/connection/message.rs

```diff
@@ -43,25 +43,22 @@ pub(super) enum Request {
 
     ServersAll,
 
+    DatabasesAll,
+    DatabaseGet { database_name: String },
     DatabasesContains { database_name: String },
     DatabaseCreate { database_name: String },
-    DatabaseGet { database_name: String },
-    DatabasesAll,
 
+    DatabaseDelete { database_name: String },
     DatabaseSchema { database_name: String },
     DatabaseTypeSchema { database_name: String },
-    DatabaseDelete { database_name: String },
 
     Transaction(TransactionRequest),
 
     UsersAll,
-    UsersContain { username: String },
-    UsersCreate { username: String, password: String },
-    UsersDelete { username: String },
-    UsersGet { username: String },
-    UsersPasswordSet { username: String, password: String },
-
-    UserPasswordUpdate { username: String, password_old: String, password_new: String },
+    UsersGet { name: String },
+    UsersContains { name: String },
+    UsersCreate { user: User },
```

Comment:
How do I update users (their passwords)?

Reply:
This is still pending an implementation in Core.